acceptancetests/assess_cloud_display.py

```diff
@@ -123,4 +124,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    sys.exit(main())
```

Comment:
main doesn't return anything, so why do this?

Reply:
Ah, I keep it in align with other scripts, and I did a bit search, found this:

http://www.artima.com/weblogs/viewpost.jsp?thread=4829

If in future we make main() to return something, sys.exit() can capture it.

Reply:
fair enough